src/components/picker/picker.tsx

```diff
@@ -252,94 +248,77 @@ export class Picker {
+            return this.renderPortal();
         }
         if (!this.chipSetEditMode) {
-            // Don't render the dropdown if the picker is not in edit mode.
-            return;
+            // render invisible dropdown if the picker is not in edit mode.
+            return this.renderPortal();
         }
 
-        const boundingRect = this.element.getBoundingClientRect();
-
         if (this.loading) {
-            return (
-                <div
-                    style={{
-                        width: `${boundingRect.width}px`,
-                    }}
-                    class={`
-                        dropdown--spinner
-                        mdc-elevation-transition
-                        mdc-elevation--z4
-                        mdc-menu-surface
-                        mdc-menu-surface--open
-                    `}
-                    tabindex="-1"
-                >
-                    <limel-spinner />
+            const spinner = (
+                <div style={{ 'padding-top': '6px', 'text-align': 'center' }}>
+                    <limel-spinner class={'dropdown--spinner'} />
                 </div>
             );
+            return this.renderPortal(spinner);
         }
 
         if (!this.items || !this.items.length) {
             if (!this.emptyResultMessage) {
-                return;
+                return this.renderPortal();
             }
-
-            return (
-                <div
+            const emptyMessage = (
+                <p
                     style={{
-                        width: `${boundingRect.width}px`,
+                        color: 'var(--lime-light-grey, #{$lime-light-grey})',
+                        'text-align': 'center',
                     }}
-                    class={`
-                        dropdown--list
-                        mdc-elevation-transition
-                        mdc-elevation--z4
-                        mdc-menu-surface
-                        mdc-menu-surface--open
-                    `}
-                    tabindex="-1"
                 >
-                    <p class="empty-result-message">
-                        {this.emptyResultMessage}
-                    </p>
-                </div>
+                    {this.emptyResultMessage}
+                </p>
             );
+            return this.renderPortal(emptyMessage);
         }
 
         const hasIcons = this.items.some(item => {
             return 'icon' in item && !!item.icon;
         });
+        const limelList = (
+            <limel-list
+                badgeIcons={hasIcons}
+                onChange={this.handleListChange}
+                type="selectable"
+                items={this.items}
+            />
+        );
+
+        if (!this.displayFullList && this.items.length > ITEM_LIMIT_NO_SCROLL) {
+            const styling = {
+                'max-height': '250px',
+                'overflow-y': 'auto',
```

Comment:
The `overflow-y` property made the shadows around the dropdown disappear so I removed it. The scrolling should be handled by the list anyway so the property should not be needed here